server/personality/personality.controller.ts

```diff
@@ -28,6 +28,7 @@ export class PersonalityController {
 
     @Get("api/personality")
     async listAll(@Query() query) {
+        console.log(query)
```

Comment:
Let's remove the `console.log`. They are useful for development but shouldn't go to production